app/mailers/user_mailer.rb

```diff
@@ -19,7 +19,7 @@ def send_password_recovery_instructions(user, from_panel = false)
     mail to: user.email, subject: 'Pedido de Recuperação de Senha', content_type: 'text/html'
   end
 
-  def send_user_random_password(user, password)
+  def welcome(user, password = nil)
```

Comment:
sim, o email agora é de boas vindas, só vou enviar a senha se for marcado no formulário de criação do usuário q deve mandar uma senha aleatória